acceptance/tests/snapshot-agent/snapshot_agent_vault_test.go

```diff
@@ -40,28 +41,141 @@ func TestSnapshotAgent_Vault(t *testing.T) {
 	// Now fetch the Vault client so we can create the policies and secrets.
 	vaultClient := vaultCluster.VaultClient(t)
 
-	vault.CreateConnectCAPolicy(t, vaultClient, "dc1")
+	// -------------------------
+	// PKI
+	// -------------------------
+	// Configure Service Mesh CA
```

Comment:
I hate to be the grammar police, but all comments need to end with proper punctuation, could you run through and double check them? Thanks!

```suggestion
	// Configure Service Mesh CA.
```

Reply:
Just to add I suppose this needs to be updated on every file sorry! :(

Reply:
Is this used in a doc string or something?

Reply:
No I don't think it parsed anywhere but it's part of our code style.

Reply:
Some context:
This is not a comment to explain the logic as we would see in comments in source code.

This comment in the test file is really just `there is so much manual set up related to vault that I put these headers in here so its easier to follow`.

So proceeding to put periods in this type of comment seems a little mismatched with the original intention of the code style.

I think I'll either move this code to helper functions or something else to break them up...or we revisit the need for an ending period for a comment that is not a sentence and is more of header.

Reply:
Yeha I think that is totally reasonable to not have periods on blocks like this. Just the other comments for the code!